app.Makefile

```diff
@@ -41,6 +41,7 @@ endef
 .PHONY: .build/app/dev
 .build/app/dev: .build/var/MARKETPLACE_TOOLS_TAG \
               | .build/app
+	$(call print_target, $@)
```

Comment:
Can we add a comment about what this line does?

Reply:
This is used throughout all of the various Makefiles (Makefile, app.Makefile, marketplace.Makefile).
I'll add a comment where `print_target` is defined in common.Makefile, saying that `print_target` displays a highlighted (yellow) message indicating the target that is being built.

Reply:
Okay, I couldn't add a comment where `print_target` is defined, since that's in the Google marketplace tools submodule. So I added a comment in Makefile and app.Makefile at the first occurrence of `print_target` in those files.